scripts/run_structure_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dir=$(dirname $0)
+
+${dir}/build.sh
+
+gcloud container builds submit --config tomcat/target/cloudbuild/structure.yaml .
```

Comment:
I would recommend adding the structure test step to the `build.yaml` build script that you're using to build the image, so that you can run both with a single `gcloud container builds submit` invocation.